test/e2e/data/infrastructure-azure/v1beta1/bases/cluster-with-kcp.yaml

```diff
@@ -1,143 +0,0 @@
----
```

Comment:
Isn't this file still referred to by other test templates, like `cluster-template-kcp-remediation`?